dashboard/state_aggregator.py

```diff
@@ -87,10 +116,75 @@ async def get_workers(self) -> dict:
             result[data["worker_id"]] = data
         return result
 
-    def _message_to_dict(self, *, message, fields_to_decode) -> dict:
+    async def get_tasks(self) -> dict:
+        # Need to wait until at least the head node stub is available.
+        await dashboard_utils.wait_for_stub(self._raylet_stub)
+
+        async def get_task_info(stub):
+            reply = await stub.GetTasksInfo(
+                node_manager_pb2.GetTasksInfoRequest(),
+                timeout=DEFAULT_RPC_TIMEOUT,
+            )
+            return reply
+
+        replies = await asyncio.gather(
+            *[get_task_info(stub) for stub in self._raylet_stub.values()]
+        )
```

Comment:
why do we need `get_task_info`? can't you just:
`[stub.GetTasksInfo(...) for stub in self._raylet_stub.values()]`?